civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -4557,7 +4557,10 @@ function _getManagerContacts($contact_id) {
     // If Leave approver is find, assign him as the manager (add contact ID to $assigned_manager_contact_ids array)
     foreach ($contactRelationships as $key => $relation) {
         if ($relation['relation'] == variable_get('relationship_name_to_use', 'has Leave Approved by')) {
-            if($relation['is_active'] == 1){
+            if($relation['is_active'] == 1
```

Comment:
And are you sure these date comparisons are always going to work fine? We'd be more safe having something like strtotime($relation['end_date']) >= time()